doc/classes/TranslationDomain.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TranslationDomain" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A set of translatable messages.
+	</brief_description>
+	<description>
+		[TranslationDomain] is a set of translatable messages. Translations can be added to or removed from it.
```

Comment:
Isn't Translation a set of translatable messages? TranslationDomain is one level higher.

Reply:
Haha, yes. I must have been too sleepy when I wrote that sentence.

What do you think about changing it to

> A collection of domain-specific [Translation]s.

Any other suggestions?

Reply:
"A self-contained collection of [Translation] resources."?